src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -78,6 +78,22 @@
         return this.requireValue(Keys.HEAD_ROTATION).asMutable();
     }
 
+    /**
+     * {@link Keys#EYE_HEIGHT}
+     * @return The height of the eyes
+     */
+    default Value.Immutable<Double> eyeHeight() {
```

Comment:
Only do Value.

I now remember the discussion ages ago. Getting a Value doesn't mean setting the `value` will actually set it back on the data holder. You have to offer it back. A DataProvider in common can reject it.

Nothing should stop your value from not being written to how you like though.

@gabizou

Reply:
Fixed